Create Sanjay Soldering

sanjay-nagesh opened the pull request:
## Describe your project in 1-3 sentences

I will be to constructing a cube-shaped object with a 4x4 array of Light-Emitting Diodes (LEDs). This LED 4x4 cube can be used for various purpose like display, decoration, etc. It will involve soldering and programming to control the LEDs and bring the cube to life.

<!-- I am building a LED light display that can be controlled through an app. -->

## Hack Club Slack handle

<!-- Join the Hack Club Slack at https://hackclub.com/slack - please write your handle ABOVE this line :)-->

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

sanjay-nagesh commented:
slack handle: sanjay

bellesea commented:
heyy thanks so much for making a PR! we're back on tuesday and will start reviewing PRs in the order that they came on. 

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget?)_

our Jan 15 11:59pm ET deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

sanjay-nagesh commented:
>from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget? 

The ESP8266s is used for practicing soldering and the addressable LED strip is for soldering project and hanging it up on my room

bellesea commented:
> > from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget?
> 
> The ESP8266s is used for practicing soldering and the addressable LED strip is for soldering project and hanging it up on my room

@sanjay-nagesh practicing soldering is SUPER awesome and a great idea but this grant is technically for funding hardware/electronics based projects (to some extent i think your project counts) so it'd be really awesome if we could see a little more of that? like perhaps you could research using the ESP9266s to connect your lights to a web app so you can control it from your phone, allowing you to both practice soldering while building smth :)

sanjay-nagesh commented:
> > > from a first glance of your project i'm wondering: what's the purpose of the esp8266s and what do you plan to do with the addressable LED strip in your budget?
> > 
> > 
> > The ESP8266s is used for practicing soldering and the addressable LED strip is for soldering project and hanging it up on my room
> 
> @sanjay-nagesh practicing soldering is SUPER awesome and a great idea but this grant is technically for funding hardware/electronics based projects (to some extent i think your project counts) so it'd be really awesome if we could see a little more of that? like perhaps you could research using the ESP9266s to connect your lights to a web app so you can control it from your phone, allowing you to both practice soldering while building smth :)

Hey!

I just changed my project to a LED cube project if that's okay!

I have updated the files accordingly 

Thanks,
Sanjay Nagesh

bellesea commented:
@sanjay-nagesh awesome!! just to check, are you referencing any guides of sorts and do you plan to use the esp8266 as your microcontroller?

sanjay-nagesh commented:
> @sanjay-nagesh awesome!! just to check, are you referencing any guides of sorts and do you plan to use the esp8266 as your microcontroller?

I am referencing a guide and yes

bellesea commented:
cool! can you please add the guide to your file? also, a heads up that we've started reviewing projects but due to the flurry last weekend (100+ PRs ksdjhfkjdshf) it might take a while before we review yours, thank you for your patience!

sanjay-nagesh commented:
> cool! can you please add the guide to your file? also, a heads up that we've started reviewing projects but due to the flurry last weekend (100+ PRs ksdjhfkjdshf) it might take a while before we review yours, thank you for your patience!

I don't have a specific guide but I might just use https://www.youtube.com/watch?v=K33bRo2tIYg&t=419s

sanjay-nagesh commented:
I think I will review my whole project and make another final commit because I want to add some parts.

alhardwarehyde reviewed (approved):
Working with LEDs is a lot of fun! I'm also a big fan of soldering so hope you enjoy!

@hackclub/bank-ops for their approval

hackliv reviewed (approved):
@sanjay-nagesh this sounds so cool! I'd love to hear more about your plans for this 😁 dming you now on Slack to chat about next steps for the grant!

bellesea commented on sanjay-nagesh.md:
```diff
@@ -34,16 +34,17 @@ What materials will you need for your project? Where will you get them? How much
+| Resistor | https://a.co/d/c4RxLoj | $7.99
+| Wood   | https://a.co/d/bpX8hm9 | $11.95
+| Wire | https://a.co/d/bpX8hm9 | $16.95
+| DOT NFC CARD |https://dotcards.net/checkouts/c/5b2e6084fd441f66c1f4fffbd2dbe0f0/shipping | $44.10 *shipping included* |
```
i think you linked the checkout instead of the product? also, are you referencing any guides for this? thanks!

sanjay-nagesh replied:
https://www.youtube.com/watch?v=K33bRo2tIYg&t=419s 
https://dotcards.net/products/dot-bundle
